resolwe/elastic/pagination.py

```diff
@@ -0,0 +1,58 @@
+""".. Ignore pydocstyle D400.
+
+==================
+Elastic Paginators
+==================
+
+Paginator classes used in Elastic app.
+
+
+.. autoclass:: resolwe.elastic.pagination.LimitOffsetPostPagination
+
+"""
+
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from rest_framework.pagination import LimitOffsetPagination, _positive_int
+
+
+def get_query_param(request, key):
```

Comment:
Is this the same method as in the base viewset? Can we have this only in one place?